src/components/Input.js

```diff
@@ -6,15 +6,15 @@ export class Input extends React.Component {
   render () {
     return (
       <input
-        placeholder={this.props.defaultValue}
+        placeholder="Program Name"
         style={[
           styles.forms.base,
           styles.forms[this.props.kind],
           this.props.style
         ]}
-          type="text"
-          value={this.state.value}
-          onChange={this.handleChange}
+        type="text"
+        value={this.state.value}
+        onChange={this.handleChange}
```

Comment:
`this.state.value` and `this.handleChange` are not defined... did you mean to delete these?